### PR TITLE
Support packaging as Zip file

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

File: DeepSpeech.py [hunk 1]
```diff
@@ -787,7 +787,7 @@ def export():
     outputs['metadata_alphabet'] = tf.constant([Config.alphabet.serialize()], name='metadata_alphabet')
 
     if FLAGS.export_language:
-        outputs['metadata_language'] = tf.constant([FLAGS.export_language.encode('ascii')], name='metadata_language')
+        outputs['metadata_language'] = tf.constant([FLAGS.export_language.encode('utf-8')], name='metadata_language')
```
Comment (reuben) [inline DeepSpeech.py]: Why?

File: DeepSpeech.py [hunk 2]
```diff
@@ -854,6 +854,26 @@ def do_graph_freeze(output_file=None, output_node_names=None, variables_blacklis
     except RuntimeError as e:
         log_error(str(e))
 
+def package_zip():
+    # --export_dir path/to/export/LANG_CODE/ => path/to/export/LANG_CODE.zip
+    export_dir = os.path.join(os.path.abspath(FLAGS.export_dir), '') # Force ending '/'
+    zip_filename = os.path.dirname(export_dir)
+
+    with open(os.path.join(export_dir, 'info.json'), 'w') as f:
+        f.write(json.dumps({
```
Comment (reuben) [inline DeepSpeech.py]: nit: `json.dump({...}, f)`

File: DeepSpeech.py (hunk 1 above)
Comment (lissyx) [reply DeepSpeech.py]: Because french ;) « Français »

Comment (reuben) [reply DeepSpeech.py]: Ah, I was thinking of language codes like fr-FR.

Comment (lissyx) [reply DeepSpeech.py]: Yeah, that's open to discussion, I'd like to expose a nice name in the JSON. If we keep `metadata_language` for language code, then we need a new flag in `util/flags.py`. Your call :)

File: DeepSpeech.py (hunk 2 above)
Comment (lissyx) [reply DeepSpeech.py]: Done, thanks

**Pull request merged**